modules/ROOT/partials/install/upgrading-info.adoc

```diff
@@ -103,10 +111,10 @@ NOTE: To simplify the upgrade process to future versions of {productname}:
 
 . Host the {productname} customizations outside of the `{prodnamecode}/` directory.
 . Update your {productname} configuration as required:
- ** Set the location of content CSS customizations using link:{baseurl}/configure/content-appearance/#content_css[`content_css`].
- ** Set the location of custom plugins using link:{baseurl}/configure/integration-and-setup/#external_plugins[`external_plugins`].
- ** Set the location of custom icon packages using link:{baseurl}/configure/editor-appearance/#icons_url[`icons_url`] instead of `icons`.
- ** Set the location of custom localization packages using link:{baseurl}/configure/localization/#language_url[`language_url`] instead of `language`.
- ** Set the location of custom skin packages using link:{baseurl}/configure/editor-appearance/#skin_url[`skin_url`] instead of `skin`.
- ** Set the location of custom themes using link:{baseurl}/configure/editor-appearance/#theme_url[`theme_url`] instead of `theme`.
+ ** Set the location of content CSS customizations using xref:configure/content-appearance.adoc#content_css[`content_css`].
+ ** Set the location of custom plugins using xref:configure/integration-and-setup.adoc#external_plugins[`external_plugins`].
+ ** Set the location of custom icon packages using xref:configure/editor-appearance.adoc#icons_url[`icons_url`] instead of `icons`.
+ ** Set the location of custom localization packages using xref:configure/localization.adoc#language_url[`language_url`] instead of `language`.
+ ** Set the location of custom skin packages using xref:configure/editor-appearance.adoc#skin_url[`skin_url`] instead of `skin`.
+ ** Set the location of custom themes using xref:configure/editor-appearance.adoc#theme_url[`theme_url`] instead of `theme`.
 ____
```

Comment:
I can't leave a suggestion sorry, but this admonition isn't rendering correctly. It needs to be converted to a block admonition like so:
```
[NOTE]
====
To simplify the upgrade process to future versions of {productname}:
. Host the {productname} customizations outside of the `{prodnamecode}/` directory.
. Update your {productname} configuration as required:
 ** Set the location of content CSS customizations using xref:configure/content-appearance.adoc#content_css[`content_css`].
 ** Set the location of custom plugins using xref:configure/integration-and-setup.adoc#external_plugins[`external_plugins`].
 ** Set the location of custom icon packages using xref:configure/editor-appearance.adoc#icons_url[`icons_url`] instead of `icons`.
 ** Set the location of custom localization packages using xref:configure/localization.adoc#language_url[`language_url`] instead of `language`.
 ** Set the location of custom skin packages using xref:configure/editor-appearance.adoc#skin_url[`skin_url`] instead of `skin`.
 ** Set the location of custom themes using xref:configure/editor-appearance.adoc#theme_url[`theme_url`] instead of `theme`.
====

Reply:
Done